config/track.go

```diff
@@ -7,7 +7,7 @@ import (
 
 var defaultIgnorePatterns = []string{
 	".*[.]md",
-	"[.]solution[.]json",
+	"solution[.]json",
```

Comment:
Is there any drawbacks to using `[.]?solution[.]json` to ensure both old and new solution files are ignored?